source/ed/tests/osm2ed_test.cpp

```diff
@@ -67,3 +70,53 @@ BOOST_AUTO_TEST_CASE(osm_poi_id_relation_should_match_mimirs_one) {
     OsmPoi osm_poi(OsmObjectType::Relation, 123456);
     BOOST_CHECK_EQUAL(osm_poi.uri, "poi:osm:relation:123456");
 }
+
+// Check if the administration is accepted if the "end_date" tag is not given
+BOOST_AUTO_TEST_CASE(osm_admin_no_end_date) {
+    OSMCache cache(std::unique_ptr<Lotus>(), boost::none);
+    ReadRelationsVisitor relations_visitor(cache, false);
+
+    Tags tags = {{"name", "NavitiaCity0"}, {"admin_level", "9"}, {"boundary", "administrative"}};
+    References ref(1, Reference());
+
+    relations_visitor.relation_callback(5, tags, ref);
+    BOOST_CHECK_EQUAL(relations_visitor.cache.admins.find(5) == relations_visitor.cache.admins.end(), false);
```

Comment:
```suggestion
    BOOST_CHECK_NE(relations_visitor.cache.admins.find(5), relations_visitor.cache.admins.end());
```

This applies for the following assertions as well ;)

Reply:
Done